src/main/java/frc/robot/Robot.java

```diff
@@ -20,19 +20,23 @@
  * project.
  */
 public class Robot extends TimedRobot {
+  
   private Command m_autonomousCommand;
 
   private RobotContainer m_robotContainer;
   private Command driveWithJoy;
+
   private static DriveSystem driveSystem = Factory.getDrive();
 
   /**
    * This function is run when the robot is first started up and should be used for any
    * initialization code.
    */
   @Override
-  public void robotInit() {
+  public void robotInit() {   
+
```

Comment:
RobotContainer should be instantiated before the drive with joystick command to avoid a null pointer in reference to the joystick.

Reply:
fixing in merge